### PR TITLE
Update threadpool.rst

### PR DESCRIPTION
**Pull request by almosnow**
From here: https://github.com/libuv/libuv/issues/145

Just so that someone won't ask the same thing again.

### CODE REVIEW

**Comment by bnoordhuis** [comment]
LGTM but can you amend the commit log per the guidelines in CONTRIBUTING.md?

**Comment by almosnow** [comment]
Check it out :ok_hand:

File: docs/src/threadpool.rst [hunk 1]
```diff
@@ -12,7 +12,11 @@ Its default size is 4, but it can be changed at startup time by setting the
 ``UV_THREADPOOL_SIZE`` environment variable to any value (the absolute maximum
 is 128).
 
-The threadpool is global and shared across all event loops.
+The threadpool is global and shared across all event loops. When a particular
+function makes use of the threadpool (i.e. when using uv_queue_work) libuv
```
Comment (saghul) [inline docs/src/threadpool.rst]: Use :c:func:`uv_queue_work` so Sphix creates a link automagically.

**Comment by saghul** [comment]
The "subsystem" for the commit log should be "doc".

**Comment by almosnow** [comment]
@saghul There it is and thanks for the reference about the threadpool handle. If everything's fine and you're going to merge, could it please be @bnoordhuis the one that does the commit? It would be nice for me.

File: docs/src/threadpool.rst [hunk 2]
```diff
@@ -12,7 +12,11 @@ Its default size is 4, but it can be changed at startup time by setting the
 ``UV_THREADPOOL_SIZE`` environment variable to any value (the absolute maximum
 is 128).
 
-The threadpool is global and shared across all event loops.
+The threadpool is global and shared across all event loops. When a particular
+function makes use of the threadpool (i.e. when using :c:func:uv_queue_work)
```
Comment (saghul) [inline docs/src/threadpool.rst]: The correct syntax is:

~~~~
:c:func:`uv_queue_work`
~~~~

Note the backticks.

**Comment by saghul** [comment]
> If everything's fine and you're going to merge, could it please be @bnoordhuis the one that does the commit? It would be nice for me.

Sure, after that last nit Ben will do the honors :-)

**Comment by almosnow** [comment]
Ready.

**Comment by saghul** [comment]
LGTM :+1:

**Comment by bnoordhuis** [comment]
@almosnow I don't want to be that guy but... can you sign the commit with your real name?  Thanks!

**Comment by almosnow** [comment]
Ok

[Definitely learning a lot about git/github today]

**Comment by bnoordhuis** [comment]
Thanks Alex, landed in ea5f1f9!

**Pull request closed without merging**